index.html

```diff
@@ -0,0 +1,89 @@
+  <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <script src="https://kit.fontawesome.com/adf6e8ca6b.js" crossorigin="anonymous"></script>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="shortcut icon" href="images/title-ai.png">
+  <title>Machin Learning Course</title>
+</head>
+<body>
+  <header>
+    <nav class="nav1">
+              <a class="nav-link" href="#">Logout</a>
+              <a class="nav-link" href="#">My Page</a>
+              <a class="nav-link" href="#">English</a>
+              <a name="twitter" class="nav-link" href="#"><i id="twitter-icon" class="fa fa-twitter text-white"></i></a>
+              <a name="facebook" class="nav-link" href="#"><i id="facebook-icon" class="fa fa-facebook text-white"></i></a>
+      </nav>
+    </nav>
+    <nav class="nav2">
+      <div class="nav2Content">
+        <a id="bordered-link" class="nav-item" href="#">AI Compaign</a>
+        <a class="nav-item" href="#">News</a>
+        <a class="nav-item" href="#sponser">Sponser</a>
+        <a class="nav-item" href="#">Join</a>
+        <a class="nav-item" href="#mainProgram">Program</a>
+        <a class="nav-item" href="about.html">About</a>
+      </div>
+      <div class="navLogo"><a href="#"><img src="images/Logo.png" alt="logo"></a></div>
+    </nav>
+  </header>
+  <section class="opening">
+    <button class="mobile-menu" type="menu" ><img src="https://img.icons8.com/ios/50/000000/menu--v4.png"/></button>
+      <span class="navbar-toggler-close"></span>
+    </button>
+    <div class="mobile-menu-list">
+      <a class="nav-item" href="about.html">About</a>
+      <a class="nav-item" href="#mainProgram">Program</a>
+      <a class="nav-item" href="#">Join</a>
+      <a class="nav-item" href="#sponser">Sponser</a>
+      <a class="nav-item" href="#">News</a>
+      <a class="nav-item" href="#">AI Compaign</a>
+    </div>
+    <h2 class="helloText">"Hello! AI World"</h2>
+    <h1 class="title">MACHIN LEARNING<br>ONLONE COURSES<br>2022</h1>
+    <p class="textBox">“Artificial Intelligence, deep learning, machine learning... whatever you are doing if you don’t understand it, learn it. Because otherwise you are going to be a dinosaur within 3 years.” <br> ~Mark Cuban</p>
+    <p class="dateText">2022.02.07(Mon) ~ 11(Fri)</p>
+    <p class="locationText">@ AI conference</p>
+  </section>
+  <section class="mainProgram" id="mainProgram">
+    <h2 class="programTitle">Main Program</h3>
+      <div class="red-line"></div>
+    <div class="programCards">
+    </div>
```

Comment:
- [ ] The only section required to be generated dynamically is only the `featured speakers` section. So it would be nice to write the code for the `program cards` here so as to follow the requirements 👍